AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -136,6 +136,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 money-viva.jp##.floating_cookie
 getmidas.com##.cookie-contain
 suncomfort.com##.cookieHinweis
+dydo.co.jp##.cookie_dialog_box
```

Comment:
Your rule is ok, but I would add `||dydo-ghd.co.jp/check-cookie/$xmlhttprequest`.

Reply:
I see, thank you.